module/tabbed.lua

```diff
@@ -102,22 +102,21 @@ tabbed.pick_with_dmenu = function(dmenu_command)
     local list_clients = {}
     local list_clients_string = ""
     for idx, c in ipairs(t:clients()) do
-        if not c.bling_tabbed then 
```

Comment:
Wrap that part into a `if c.window ~= client.focus.window then` so that the window to which a client is added doesn't show up in the dmenu itself as that is confusing when picking